### ISSUE TITLE
0.6.26 (2020-04-29) Causing Freezing / Massive stuttering

### ISSUE THREAD

**Issue by BeeGeePi** [Kyusung4698/PoE-Overlay#690]
**Describe the bug**
Whenever I run 0.6.26 (2020-04-29) it causes my computer to constantly freeze/stutter to the point it is unusable. When it is freezing/stuttering my mouse barely responds and none of my keyboard inputs work.  I reverted to an older version of PoE-Overlay (0.6.24) and it is running fine again.  

**To Reproduce**
Run 0.6.26 (2020-04-29) on Windows 10

I also run Aqcuisition, YOLOMouse, LutBot PoE Macro, and PoE when running this version of PoE-Overlay

## 🌍  Your Environment

- OS: Windows 10
- Version: 0.6.26 (2020-04-29)
- PoE: PoE client 3.10.1f

Nvidia GTX 1070, 32 GB RAM, CPU AMD Ryzen 7 1770x

**Comment by remd** [comment]
Does it freeze your computer or just Path of Exile?

I think am seeing this issue on 0.6.28 but in the form of FPS drops in Path of Exile itself. I'm able to cause FPS drops by moving the mouse quickly back and forth.

The problem completely disappears once I exit PoE-Overlay.

**Comment by Darkblitz9** [comment]
So I found out it's Yolomouse. I started suspecting it once I noticed that the hanging/crashes occur when doing right clicks on standard windows menus. I only discovered the issue exists because I took the time to slowly trudge through my comp operating at like 1 FPS for 10 minutes or so while I slowly killed off processes in Task Manager one by one. As soon as I killed Yolomouse, everything started running fine again. Find another cursor replacer. I know I will.
